shard-operator-parent/shard-operator-v1/pom.xml

```diff
@@ -2,19 +2,28 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
   <parent>
     <groupId>com.redhat.service.smartevents</groupId>
-    <artifactId>build-parent</artifactId>
+    <artifactId>shard-operator-parent</artifactId>
     <version>0.0.1-SNAPSHOT</version>
-    <relativePath>../build-parent/pom.xml</relativePath>
   </parent>
 
-  <modelVersion>4.0.0</modelVersion>
-  <artifactId>shard-operator</artifactId>
+  <artifactId>shard-operator-v1</artifactId>
   <version>0.0.1-SNAPSHOT</version>
-  <name>SmartEvents :: Shard Operator</name>
+  <name>SmartEvents :: Shard Operator :: v1</name>
 
   <dependencies>
+    <dependency>
+      <groupId>com.redhat.service.smartevents</groupId>
+      <artifactId>shard-operator-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.guava</groupId>
+      <artifactId>guava</artifactId>
+    </dependency>
```

Comment:
Are you adding `guava` dependency to the project or it was already used?
Where is it used?

Reply:
It was already [used](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/e396760c1cdf9e9fc8219694c7d5248459ec898a/shard-operator-parent/shard-operator-v1/src/main/java/com/redhat/service/smartevents/shard/operator/v1/resources/BridgeExecutor.java#L8).

When I tidied up the dependencies for the new `-v1` module I had to add back an explicit dependency.

I assume it was a transient dependency elsewhere before.

Reply:
Ok thanks for the clarification, if that is the only use I think we should remove it.
Created a [ticket to review it](https://issues.redhat.com/browse/MGDOBR-1222).